sources/platform/runtime/machine-image/ami-v662.md

```diff
@@ -0,0 +1,217 @@
+page_main_title: v6.6.2
+main_section: Platform
+sub_section: Runtime
+sub_sub_section: Machine image
+page_title: Description of what is available in Machine Image v6.6.2
+page_description: A complete list of language versions, Docker versions, packages and tools available in Machine Image v6.6.2
+page_keywords: CI/CD, shippable CI/CD, documentation, shippable, config, yml, AMI, Docker
+
+# Machine image v6.6.2 (Docker TAG v6.6.2)
+
+**Release Date: june 28, 2018**
+
+**What is installed on the machine image?**
+
+## u14 sku
+* Host Operating System: Ubuntu 14.04 LTS
+* Build Container Operating System: Ubuntu 16.04.3 LTS
+* Kernel Version: 3.13.0-125-generic
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: aufs
+* Root Dir: /data/aufs
+* Backing Filesystem: extfs
+* Dirperm1 Supported: false
+* Cgroup Driver: cgroupfs
+* Shippable Official u14 and u16 Docker Images with TAG: `v6.6.2`
+
+## u16 sku
+
+* Host Operating System: Ubuntu 16.04 LTS
+* Build Container Operating System: Ubuntu 16.04.3 LTS
+* Kernel Version: 4.4.0-1049-aws
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: aufs
+* Root Dir: /data/aufs
+* Backing Filesystem: extfs
+* Dirperm1 Supported: false
+* Cgroup Driver: cgroupfs
+* Shippable Official u16 Docker Images with TAG: `v6.6.2`
+
+## c7 sku
+* Host Operating System: CentOS 7
+* Build Container Operating System: CentOS 7
+* Kernel Version: 3.10.0-514.10.2.el7.x86_64
+* **Docker Server Version: 17.06.0-ce**
+* Storage Driver: overlay
+* Root Dir: /data
+* Backing Filesystem: xfs
+* Cgroup Driver: cgroupfs
+* Shippable Official c7 Docker Images with TAG: `v6.6.2`
+
+## w16 sku
+* Host Operating System: Windows Server 2016 Datacenter Edition (Version 1607)
+* Build Container Operating System: Windows Server Core 2016 v10.0.14393.1884
+* Docker Server Version: 17.10.0-ce
```

Comment:
I don't think this is correct. If I remember correctly, the node init script for Windows installs `17.06.2-ee`. Can you please confirm whether this is right?

Reply:
yea docker version for w16 is 17.10.0-ce

Reply:
But w16 is not what's being documented here. This is the Docker version that's installed on the Windows host.

Reply:
It's still `17.06.2-ee-5`: https://github.com/Shippable/node/blob/master/initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1#L4